### DISCUSSION TITLE
Don't understand effect of softPWM

### DISCUSSION THREAD

**Discussion by Apehaenger** [Dlloydev/sTune#25] [Q&A] [answered]
Quite cool to see that you re-a lived the old PID/Autotune lib.

I'm trying to get my PTC+SSR autotuned with the library but unfortunately I don't understand the examples (MAX6675_PTC_SSR/sTune_QuickPID_Less_Overshoot/) and the effect of the softPWM's optimumOutput :-(

The general functionality of the softPWM is clear to me. But I don't see for what the optimumOutput result/return is used, except for plotting.
I also have no change in output nor optimumOutput during tuning, which is logic to me as there's no feedback to the empty PID controller?!

What's my understanding fault?

**Comment by Apehaenger** [comment, marked as answer]
My fault!!
Had a stupid pointer-init mistake in my code.

Sorry to have bothered you ^^

**Comment by e1isev** [comment]
I also have this problem, did you figure out what coptimumOutput is used for?
